aiohttp/web_fileresponse.py

```diff
@@ -102,6 +105,31 @@ async def _sendfile(
         await super().write_eof()
         return writer
 
+    @staticmethod
+    def _strong_etag_match(etag_value: str, etags: Tuple[ETag, ...]) -> bool:
+        if len(etags) == 1 and etags[0].value == ETAG_ANY:
+            return True
+        else:
+            return any(etag.value == etag_value for etag in etags if not etag.is_weak)
```

Comment:
Nitpick, this could be less nested:
```suggestion
        return any(etag.value == etag_value for etag in etags if not etag.is_weak)
```